ta/pkcs11/src/pkcs11_helpers.c

```diff
@@ -0,0 +1,71 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright (c) 2018-2020, Linaro Limited
+ */
+
+#include <pkcs11_ta.h>
+#include <string.h>
+#include <tee_internal_api.h>
+#include <util.h>
+
+#include "pkcs11_helpers.h"
+
+static const char __maybe_unused unknown[] = "<unknown-identifier>";
+
+struct any_id {
+	uint32_t id;
+#if CFG_TEE_TA_LOG_LEVEL > 0
+	const char *string;
```

Comment:
What is the point with `id` above if this isn't defined?

Reply:
This structure is used (later) for ID to string conversion and also for valid IDs lists.
In this P-R it is used to trace invoked commands and unused when traces are disabled.
In a later change, the ID will allow to list valid attribute IDs, or valid mechanism IDs. For those, the array will be used both for valid ID test and for (debug) string conversion.

Reply:
Fair enough